cluster/repository/cluster_repository.go

```diff
@@ -122,10 +187,15 @@ func (m *GormClusterRepository) LoadClusterByURL(ctx context.Context, url string
 // Create creates a new record.
 func (m *GormClusterRepository) Create(ctx context.Context, c *Cluster) error {
 	defer goa.MeasureSince([]string{"goa", "db", "cluster", "create"}, time.Now())
+	log.Debug(ctx, map[string]interface{}{"cluster": c}, "creating cluster record in db")
```

Comment:
we should not print all sensitive cluster info in the debug logs while creating it. how about printing cluster info which is not sensitive.

Reply:
good catch! let me remove this immediately

Reply:
10666f8